ci/test_cpp.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
```

Extra shebang can be removed.